migrations/3_configure.js

```diff
@@ -67,7 +82,7 @@ async function deploy(options, accounts) {
     // address _steward,
     steward.address,
     // address _token
-    patronageERC721.address,
+    patronageERC20.address,
```

Comment:
This bug took me an hour and a half to find :see_no_evil: Sorry if you ran into this too @DenhamPreen

Reply:
Haha I actually got stuck on 2_deploy.js